config/samples/advertise_two_neighs_two_configs.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: frrk8s.metallb.io/v1beta1
+kind: FRRConfiguration
+metadata:
+  name: test-0
+  namespace: default
```

Comment:
change "default" to "frr-k8s-system" everywhere?

Reply:
done